### PR TITLE
Update the contributors list to source data locally

### PR DESCRIPTION
**Pull request by JosephusPaye**
Implements the proposal in #207. Also makes minor simplifications to the display of contributors.

I was gonna to work on #69 but merging `main` brought up errors about the GH API token for the contributions so I switched to doing this instead.

I've added the data for the Introduction to NUbots page, and we can fill out the rest in another PR.

Preview: [Introduction to NUbots](https://deploy-preview-209--nubook.netlify.app/team/introduction)

### CODE REVIEW

**Review by lach-j — APPROVED** [review]
Looks great :smile:

Only small issue I can see is that once the authors get added to the existing pages, the dates will reflect that commit instead of when content was added. Shouldn't really matter too much though and will fix itself eventually :shrug:

File: src/images/nubots-icon.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...-.218-.829-.287-1.336-.324-2.024-.053-.696 1.423-1.058 1.718-2.513 2.227-.622.217-1.436.466-1.81.556-.24.057-.5.098-.758.123a18 18 0 00-.1.642 9.161 9.161 0 003.455-.979c1.12-.555 1.617-.668 2.147-.488l.103.034a14.709 14.709 0 01-8.327 2.667l-.133.002c-2.34 0-4.56-.551-6.542-1.534v-.007c-.018-.664-.416-2.11-.735-2.674-.374-.659-1.336-1.788-2.911-3.418a21.06 21.06 0 00-.938-.927c.094.147.186.294.27.433.333.555.836 1.29 1.118 1.63.983 1.19 1.07 1.42 1.07 2.817v.863a15.731 15.731 0 01-5.516-7.275l.473-.016-.397-.173-.172-.075a16.139 16.139 0 01-.912-5.365 16.214 16.214 0 01.961-5.504A15.733 15.733 0 017.954 3.53a14.726 14.726 0 018.883-2.28l.256.16c.338.213.803.46 1.033.55.09.033.242.092.429.164-.14-.181-.27-.366-.379-.543l-.124-.202a14.868 14.868 0 017.167 3.17c.51.413.995.86 1.45 1.336a15.82 15.82 0 013.154 4.82 4.57 4.57 0 01-.116.374c-.176.508-.173.534.125 1.739.362 1.457.462 3.226.26 4.576-.062.404-.09.84-.075 1.063.393-2.272.55-2.955.812-3.558l.086-.199a16.244 16.244 0 01-.066 4.912 15.968 15.968 0 01-1.55 4.687zM15.877.772C7.262.772.253 8.024.253 16.94c0 8.914 7.01 16.166 15.624 16.166 8.616 0 15.625-7.252 15.625-16.166 0-8.915-7.01-16.167-15.625-16.167z" fill="#F9A50D"/></svg>
```
Comment (ysims) [inline src/images/nubots-icon.svg:1]: How come this has been added? I don't see a nubots icon in the contributors/last edited bar?

**Review by ysims — COMMENTED** [review]
You've got a `yarn.lock` file in your changes.

File: src/images/nubots-icon.svg (hunk 1 above)
Comment (JosephusPaye) [reply src/images/nubots-icon.svg:1]: It's used for authors without a GitHub profile, as the default avatar. E.g. if someone like Stephan (assuming he doesn't have a GH account) were to add to a page they can be credited with just their name, and the NUbots icon will be used, without linking to GitHub.

**Comment by JosephusPaye** [comment]
The `yarn.lock` change is expected: it happens whenever a dependency is added, removed, or upgraded. In this case the change is due to removing the `gatsby-source-graphql` package, which was used for talking to the GitHub API for commits and contributors.

As to why we even have lock files in the first place, [see this article](https://snyk.io/blog/what-is-package-lock-json/).

File: src/images/nubots-icon.svg (hunk 1 above)
Comment (ysims) [reply src/images/nubots-icon.svg:1]: Ah, ok.

**Review by ysims — COMMENTED** [review]
Can't really comment on the js but I can't see any issues. 
I'm gonna agree with the approach since even though it is more manual, it is much more correct. I don't really want to be credited for a page I only reformatted or renamed :sweat_smile:.

**Review by ysims — APPROVED** [review]
:+1:

**Pull request merged**